packages/remix-dev/config.ts

```diff
@@ -263,12 +263,6 @@ export interface RemixConfig {
    */
   serverBuildPath: string;
 
-  /**
-   * The target of the server build. Defaults to "node-cjs".
-   *
-   * @deprecated Use a combination of `{@link AppConfig.publicPath}`, `{@link AppConfig.serverBuildPath}`, `{@link AppConfig.serverConditions}`, `{@link AppConfig.serverDependenciesToBundle}`, `{@link AppConfig.serverMainFields}`, `{@link AppConfig.serverMinify}`, `{@link AppConfig.serverModuleFormat}` and/or `{@link AppConfig.serverPlatform}` instead.   */
-  serverBuildTarget?: ServerBuildTarget;
-
```

Comment:
Since we're exporting `RemixConfig` type (as `ResolvedRemixConfig`), we should keep this until the next major release, as removing it would be a breaking change otherwise.
https://github.com/remix-run/remix/blob/837517ef13e6e52acaea7adc8d2c1648aed11ae9/packages/remix-dev/index.ts#L3

```suggestion
  /**
   * The target of the server build. Defaults to "node-cjs".
   *
   * @deprecated Use a combination of `{@link AppConfig.publicPath}`, `{@link AppConfig.serverBuildPath}`, `{@link AppConfig.serverConditions}`, `{@link AppConfig.serverDependenciesToBundle}`, `{@link AppConfig.serverMainFields}`, `{@link AppConfig.serverMinify}`, `{@link AppConfig.serverModuleFormat}` and/or `{@link AppConfig.serverPlatform}` instead.
   */
  serverBuildTarget?: ServerBuildTarget;

```